### ISSUE TITLE
Notifications don't get enabled in some edge cases

### ISSUE THREAD

**Issue by jaredwolff** [pyrinas-iot/pyrinas-ble-client#1]
Mostly related to the release bonds button. 

```
arning> ble_m_periph: Unable to write. Notifications not enabled!
<info> app: Core 575 Ambient 22 Status 3
<warning> ble_m_periph: Unable to write. Notifications not enabled!
<info> app: Core 575 Ambient 21 Status 3
<warning> ble_m_periph: Unable to write. Notifications not enabled!
<info> app: Core 575 Ambient 21 Status 3
<warning> ble_m_periph: Unable to write. Notifications not enabled!
<info> app: Core 580 Ambient 22 Status 3
```

**Comment by jaredwolff** [comment]
It has something to do with the initial bonding process. Subsequent resets seem to be ok..

**Comment by jaredwolff** [comment]
Had to do with the peer manager taking *a little* longer than normal to finish. The command was issued but the connection wasn't done setting up. Popped in a fix here: d1eac7a5d51cad381c4e3efb653a72d0517863f4
